modules/eco-setting-node-maintenance-cli.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+//nodes/nodes/eco-node-maintenance-operator.adoc
+
+:_content-type: PROCEDURE
+[id="eco-setting-node-maintenance-cli_{context}"]
+= Setting a node to maintenance mode in the CLI
+
+Set a node to maintenance mode by marking it as unschedulable and using the `oc adm drain` command to evict or delete pods from the node.
+
+.Procedure
+
+. Mark the node as unschedulable. The node status changes to `NotReady,SchedulingDisabled`.
++
+[source,terminal]
+----
+$ oc adm cordon <node1>
+----
+
+. Drain the node in preparation for maintenance.
++
+[source,terminal]
+----
+$ oc adm drain <node1> --delete-local-data --ignore-daemonsets=true --force
+----
+
+* The `--delete-local-data` flag removes any virtual machine instances on the node that use `emptyDir` volumes. Data in these volumes is ephemeral and is safe to be deleted after termination.
+
+* The `--ignore-daemonsets=true` flag ensures that daemon sets are ignored and pod eviction can continue successfully.
```

Comment:
is it DaemonSets ? Do we have a standard way to define these ? 

DaemonSet and ReplicaSet

Reply:
Yes, seems to align with the guideline for [API object formatting](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting)